### DISCUSSION TITLE
Application Web UI seems to have some errors

### DISCUSSION THREAD

**Discussion by zhenjiaguo** [apache/kyuubi#1345] [Q&A] [answered]
I execute some SQL, it show different between `Kyuubi Query Engine` tab and `SQL` tab. `SQL` tab seems show incorrect.

Kyuubi Query Engine show  is correct:
![image](https://user-images.githubusercontent.com/29809822/140516863-50754f02-4786-4441-818c-eb36141eb22f.png)

SQL tab looks incorrect. Except for queries, other SQL are displayed twice:
![image](https://user-images.githubusercontent.com/29809822/140516498-2538b4b7-79ce-4411-8f82-4a55bf50c836.png)

Is this normal?

------
My environment is:
OS：Ubuntu 20.04 LTS
deploy mode: client
RELEASE file content:
```
Kyuubi 1.4.0-SNAPSHOT (git revision 1f87aec0) built for
Java 1.8.0_292
Scala 2.12
Spark 3.1.2
Hadoop 3.2
Hive 2.3.9
```

**Comment by yaooqinn** [comment, marked as answer]
this a a known spark issue

jia ***@***.***> 于2021年11月5日周五 下午9:48写道：

**Comment by wesleygoi-liftoff** [comment]
hi @zhenjiaguo how did you get this kyuubi query engine page?

**Reply by pan3793** [reply]
It's shown on the Spark Live UI of each Spark application launched by Kyuubi.
